src/application/res/app.py

```diff
@@ -22,22 +73,56 @@ def index():
 """rendering page to search in OS"""
 
 
-@app.route('/search')
-def search():
-    # response = search_os.simple_search(client = client, search_text = searchField)
-    return render_template('search.html')
-
-
-@app.route('/search/simple')
-def search_simple():
-    return os_manager.simple_search(index_name, request.args.get('searchString'))
-
-
-@app.route('/search/advanced')
-def search_advanced():
-    search_info = json.loads(urllib.parse.unquote(request.args.get('searchString')))
-    return os_manager.advanced_search(index_name, search_info)
+#@app.route('/search')
+#def search():
+#    # response = search_os.simple_search(client = client, search_text = searchField)
+#    return render_template('search.html')   
 
+@app.route('/search', methods=['GET', 'POST'])
+def search():
+    simpleSearchForm = SimpleSearchForm()
+    simpleSearchResult = ""
+    advancedSearchForm = AdvancedSearchForm()
+    advancedSearchResult = ""
+    if simpleSearchForm.validate_on_submit():
+        searchValue = simpleSearchForm.searchValue.data
+        resultTmp = os_manager.simple_search("amoscore", searchValue)
+        if len(resultTmp)>0:
+            simpleSearchResult=renderResult(resultTmp)
+        else:
+            simpleSearchResult="No results found."
+    
+    if advancedSearchForm.validate_on_submit():
+        search_info = {}
+        for entry in advancedSearchForm.entry.data:
+            parameter_name = entry['metadata_tag']
+            search_content = entry['value']
+            operator = entry['condition']
+            search_info[parameter_name] = {
+                'search_content': search_content,
+                'operator': operator
+            }
+            print(search_info)
+        resultTmp = os_manager.advanced_search(index_name="amoscore", search_info=search_info)
+        if len(resultTmp)>0:
+            advancedSearchResult=renderResult(resultTmp)
+        else:
+            advancedSearchResult="No results found."
+
+    return render_template('simpleSearch.html',simpleSearchForm=simpleSearchForm,simpleSearchResult=simpleSearchResult, advancedSearchForm=advancedSearchForm, advancedSearchResult=advancedSearchResult)
```

Comment:
Maybe we should add 1-2 lines of explanantion to this part of code. I don't think, that we would need more, because it's quite technical. What's your opinion @JMiltner97 ?

Reply:
Yes I'll make 1 or 2 comments lateron, thank you :)